src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -21,4 +51,47 @@
     public static final String NON_EMPLOYEE_TRAVEL = "Non-Employee Travel";
 
     public static final String USE_OVERSIZED_CHECK_STUB_TEXT_QUESTION_ID = "UseOversizedCheckStubTextQuestion";
+
+    public static final String ACH_INCOME_FILE_DATE_FORMAT = "yymmddHHmm";
+    public static final String ADVACE_DEPOSIT_DEFAULT_CAMPUS_CODE = "IT";
+    public static final String ADVANCE_DEPOSIT_NOTE_FILE_PREFIX = "advancedeposit_note_";
+    public static final String BATCH_DETAIL_TYPE = "Batch";
+    public static final String IU_FROM_EMAIL_ADDRESS_PARM_NM = "prod_control@cornell.edu";
```

Comment:
Good catch. Either a properties file or a parameter. A parameter is going to be easier to change. We do have some email properties (and parameters I believe) already. Let me look and see what we're doing elsewhere.